2-mandatory/6-writing-tests-advanced.js

```diff
@@ -55,6 +55,15 @@ function formatCourseworkResult(trainee) {
     score: 63
   }
 */
```

Comment:
These tests look good to me, but you might need to think about the best way to name your tests.
Imagine you make a change to your code - and a test starts to fail. In the test output, you might see `the output has trainee score` or `the output has trainee name, Aman` - but this won't give you any useful information about which part of the code might be broken. Can you think of better names for these tests?